velox/functions/prestosql/URLFunctions.h

```diff
@@ -18,6 +18,7 @@
 #include <boost/regex.hpp>
 #include <cctype>
 #include "velox/functions/Macros.h"
+#include "velox/functions/lib/string/StringImpl.h"
```

Comment:
Let's remove this include and move urlEscape and urlUnescape to this file.

Reply:
Updated.